sampleapp/src/main/java/org/wordpress/android/sampleapp/SampleMimeTypeProvider.kt

```diff
@@ -4,9 +4,10 @@ import org.wordpress.android.mediapicker.api.MimeTypeProvider
 import javax.inject.Inject
 
 class SampleMimeTypeProvider @Inject constructor() : MimeTypeProvider {
-    override val imageTypes: List<String> = listOf("JPEG", "PNG")
+    override val imageTypes: List<String> = listOf("image/jpeg", "image/png")
```

Comment:
Here we were expecting mime types, but we gave extensions, and this broke the system picker:

| Before (it didn't list files we wanted) | After |
| --- | --- |
| ![before2](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/assets/1657201/de59bed0-9696-41d0-be3a-32d82cc368f7) | ![after2](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/assets/1657201/c6fe707c-7e21-4bd2-a1b0-4aa7e618989c) |